src/containers/MoviesList/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import "./movie-list.scss"
 import { Container } from "../Container";
 import { MovieCard } from "../../components/MovieCard";
+import { Genre } from "../../components/MovieCard/interface";
```

Comment:
It would make sense to import your `movies` data as well. Having those defined within your `<MoviesList >` component's body doesn't seem to be good idea.